ch01.adoc

```diff
@@ -172,9 +174,17 @@ On the other hand identifying the vertical coordinate is not necessarily suffici
 In particular a model may output data on the parametric (usually dimensionless) vertical coordinate used in its mathematical formulation.
 To achieve the goal of being able to spatially locate all data values, this convention provides a mapping, via the **`standard_name`** and **`formula_terms`** attributes of a parametric vertical coordinate variable, between its values and dimensional vertical coordinate values that can be uniquely located with respect to a point on the earth's surface (<<parametric-vertical-coordinate>>; <<parametric-v-coord>>).
 
-It is often the case that data values are not representative of single points in time and/or space, but rather of intervals or multidimensional cells.
-This convention defines a **`bounds`** attribute to specify the extent of intervals or cells.
-When data that is representative of cells can be described by simple statistical methods, those methods can be indicated using the **`cell_methods`** attribute.
+It is often the case that data values are not representative of single points in time, space and other dimensions, but rather of intervals or multidimensional cells.
+CF defines a **`bounds`** attribute to specify the extent of intervals or cells.
+Because both the <<NUG>> and <<COARDS>> define coordinate variables but not cells or bounds, many applications assume that gridpoints are always located at the centers of their cells.
+This assumption does not hold in CF. If bounds are not provided, the location of the gridpoint within the cell is undefined, and nothing can assumed about the location and extent of the cell.
```

Comment:
missing word: and nothing can **be** assumed...

Reply:
Thanks. Done.